src/signs/ui/search/SignsSearchForm.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from 'react'
+import { stringify } from 'query-string'
+import { Button, Col, Form, FormControl, Popover, Row } from 'react-bootstrap'
+import { RouteComponentProps, withRouter } from 'react-router-dom'
+import _ from 'lodash'
+import HelpTrigger from 'common/HelpTrigger'
+import { SignQuery } from 'signs/domain/Sign'
+import replaceTransliteration from 'fragmentarium/domain/replaceTransliteration'
+
+interface Props extends RouteComponentProps {
+  signQuery: SignQuery
+  sign: string | undefined
+}
+
+function SignsSearchHelp(): JSX.Element {
+  const Section = ({ label, text }) => (
+    <li>
+      <Row>
+        <Col xs={2} className="pr-1">
+          {label}
+        </Col>
+        <Col className="pl-0">{text}</Col>
+      </Row>
+    </li>
```

Comment:
Why put a row in a list item? It seems to make no sense.

Reply:
It somehow forces the columns not to wrap.

Reply:
Lists should have nothing to do with columns or wrapping. Use a proper way of making it not-wrapping instead of relying on hacks. Not sure exactly what you are seeking to achieve but a table would provide fixed columns, or you could just set the width for the label.